app.js

```diff
@@ -228,8 +231,12 @@ async function touch(pullRequest, jiraInfo) {
 	const multiCommitMessage = (jiraInfo.numCommits === 0) ? "No commits found on PR" : (jiraInfo.numCommits === 1) ? "This PR includes exactly 1 commit!" : "This PR has " + jiraInfo.numCommits + " commits" + (multiCommitPass ? "" : "; consider squashing:");
 	const promises = [
 		github.createStatus("jira-ticket", pullRequest, jiraInfo.pass, url, jiraInfo.description),
-		github.createStatus("single-commit", pullRequest, multiCommitPass, url, multiCommitMessage)
 	];
+
+	if (!(process.env.ALLOW_MANY_COMMITS === "TRUE")) {
```

Comment:
I'd rather if you named this `DISABLE_SINGLE_COMMIT_CHECK`.  Also, please move the "multiCommitPass" and "multiCommitMessage" lines into this if block.